Chummer/customdata/Neon Anarchy House Rules/amend_qualities.xml

```diff
@@ -230,14 +250,23 @@
 				</oneof>
 			</required>
 		</quality>
-		<quality>
-			<name>Puppet Master</name>
-			<hide />
-		</quality>
-		<quality>
-			<name>Revels in Murder</name>
-			<hide />
-		</quality>
+        <quality>
+            <name>Prototype Transhuman</name>
+            <bonus amendoperation="replace">
+                <prototypetranshuman>1</prototypetranshuman>
+                <selectquality amendoperation="replace">
+                    <quality>Aged</quality>
+                    <quality>Albinism II</quality>
+                    <quality>Allergy (Common, Moderate)</quality>
+                    <quality>Astral Beacon</quality>
+                    <quality>Blank Slate</quality>
+                    <quality>Incomplete Deprogramming</quality>
+                    <quality>Insomnia (Basic)</quality>
+                    <quality>Low Pain Tolerance</quality>
+                    <quality>Sensitive System</quality>
+                </selectquality>
+            </bonus>
```

Comment:
Just to be clear, `amendoperation="replace"` completely replaces the entire existing node with the entire new node. Having another node with `amendoperation="replace"` inside of it makes no sense, and probably signals that you want to do something that may not work the way you actually want it to work.

Reply:
I believe this was solved in the latest commit.